modules.d/70kernel-network-modules/module-setup.sh

```diff
@@ -36,5 +36,8 @@ installkernel() {
         '=drivers/net/phy' \
         '=drivers/net/team' \
         '=drivers/net/ethernet' \
-        ecb arc4 bridge stp llc ipv6 bonding 8021q ipvlan macvlan af_packet virtio_net xennet
+        ecb arc4 bridge stp llc ipv6 bonding 8021q ipvlan macvlan virtio_net xennet
+
+    # arping depends on af_packet
+    hostonly='' instmods af_packet
```

Comment:
The `arping` call is in the net-lib module. So shouldn't the `af_packet` package installed there?